client/client-multi/client-c/build.sh

```diff
@@ -168,7 +108,10 @@ case "$cmd" in
         UNITTESTS_COMPILE=1
```

Comment:
Since we are following TDD we shouldn't disable unit tests during development. If you developing your feature without tests then there is something wrong with feature design. Tests must accompany every step you take. By interleaving test implementation with feature implementation you iteratively growing test suite and killing all bugs one-by-one without putting testing phase till the doomsday. 

Of course, you can disable unit testing **locally** in certain cases if you want to.

But leaving these flags in build script and thus allowing other people use bad practices could lead to dangerous consequences.